src/web.security/jwt.encryption.ts

```diff
@@ -0,0 +1,42 @@
+import * as crypto from "crypto";
+import {OAUTH2_REQUEST_KEY} from "../properties";
+import * as jose from "node-jose";
+
```

Comment:
I think this implementation might be ported from the Java equivalent version somewhere. Worth referencing (the repo?) in a general comment at the top of this file I think.